prowler/lib/outputs/common_models.py

```diff
@@ -29,7 +29,7 @@ class FindingOutput(BaseModel):
     """
 
     auth_method: str
-    timestamp: datetime
+    timestamp: Union[int, datetime]
```

Comment:
Can you add a test creating a `FindingOutput` with the unix timestamp?

Reply:
Sure!

Reply:
Done ✅